### DISCUSSION TITLE
Desktop location from gpsd

### DISCUSSION THREAD

**Discussion by Claeysson** [devemux86/cruiser#104] [Q&A] [answered]
Hi!

Is it possible to show the current location in Cruiser desktop on Linux using gpsd as data source?
I have a working gpsd daemon but the location does not show up in Cruiser. Do I need to do any configuration?

Thanks,
Martin

**Comment by devemux86** [comment, marked as answer]
@Claeysson thanks for the interesting idea!

Desktop Cruiser currently does not read location services.

I could check and implement it if there is enough interest?

**Comment by Claeysson** [comment]
It would be much appreciated by me and a great feature of a wonderful app!
